ports/open62541/portfile.cmake

```
@@ -33,7 +33,7 @@ vcpkg_cmake_configure(
     OPTIONS
         ${FEATURE_OPTIONS}
         ${OPEN62541_ENCRYPTION_OPTIONS}
-        -DOPEN62541_VERSION=${VERSION}
+        -DOPEN62541_VERSION="v${VERSION}"
```

I guess it should be
```suggestion
        "-DOPEN62541_VERSION=v${VERSION}"
```

---

Done.